scripts/scil_bundle_label_map.py

```diff
@@ -327,7 +323,7 @@ def main():
         timer = time.time()
         new_sft = StatefulTractogram.from_sft(sft.streamlines, sft_list[0])
         cut_sft = cut_streamlines_with_mask(
-            new_sft, binary_mask, min_len=min_len,
+            new_sft, binary_mask,
```

Comment:
Removed the `min_len` parameter. It did not really make sense to specify this as a parameter as it would prevent any streamlines from being saved if the labels did not span all streamline points.

@frheault please confirm.